tests/test_self.py

```diff
@@ -341,6 +341,38 @@ def test_wrong_import_position_when_others_disabled(self) -> None:
             actual_output = actual_output[actual_output.find("\n") :]
         assert self._clean_paths(expected_output.strip()) == actual_output.strip()
 
+    def test_progress_reporting(self) -> None:
+        module1 = join(HERE, "regrtest_data", "import_something.py")
+        module2 = join(HERE, "regrtest_data", "wrong_import_position.py")
+
+        expected_output = textwrap.dedent(
+            f"""
+        Using config file pylint/testutils/testing_pylintrc
+        Get ASTs.
+        tests/regrtest_data/wrong_import_position.py
```

Comment:
Should we use `module1` and `module2` here as well, just like you did with the other one?

Reply:
I don't follow, can you explain?  Do you mean add `(1)` to the end of the message, like is done during the actual linting (`x.py (1 of 2)`).  I didn't feel that a counter would be helpful, given that I don't currently have the total number of ASTs to get.

Reply:
See line 357. You're using an f-string there for the module name, should we do so here as well?

Reply:
I see what you mean, in that case the expected output string becomes

```
        Get ASTs.
        {module2}
        {module1}
```

which makes sense.  The expected_output string should be moved closer to the assert as well, that's where it's used.

Reply:
Pushed fix for this, thx.

Reply:
Leaving to y'all to resolve things as I make changes.

Reply:
Actually I'll resolve this, no need to keep extra noise in the already-long discussions :wave: